### DISCUSSION TITLE
Documentation for polaris viz

### DISCUSSION THREAD

**Discussion by sonica-sharma** [Shopify/polaris-react#6754] [Q&A] [answered]
Request you to please update/share documentation on how to use polaris viz and the input format

**Comment by alex-page** [comment, marked as answer]
Hey @sonika-rk-sharma `polaris-viz` isn't maintained by the team that works in this repository. We are working to consolidate a documentation across a lot libraries and hopefully we surface the documentation in a centralised place.
